src/features/Application/ResumeComponent.tsx

```diff
@@ -15,15 +15,17 @@ export interface IResumeProps {
 const ResumeComponent: React.FC<IResumeProps & FieldProps> = (props) => {
   const viewResume = <ViewPDFComponent {...props} />;
   return (
-    <div style={{ marginBottom: '20px' }}>
+    <Flex mb={'32px'}>
```

Comment:
I think Flex needs to be imported, it's causing a build error

Reply:
Yikes, messed up the resolve merge conflicts - was meaning to get rid of Flex